tests/profile/admin/test_profile_indicator_admin.py

```diff
@@ -47,6 +96,117 @@ def test_indicator_queryset(self, mocked_request, indicator, qualitative_indicat
         # assert count
```

Comment:
Not related to this change but this seems like a planning comment that was forgotten to be removed